src/renderer/components/+extensions/extensions.tsx

```diff
@@ -49,17 +65,38 @@ export class Extensions extends React.Component {
     }
   };
 
+  @observable
+  extensionState = observable.map<string, ExtensionState>();
+
   @observable search = "";
   @observable installPath = "";
+  
+  componentDidMount() {
+    disposeOnUnmount(this,
+      reaction(() => this.extensions, (extensions) => {
+        const removedUninstalling = getRemovedUninstalling(extensions, this.extensionState);
+
+        removedUninstalling.forEach(({ displayName }) => {
+          Notifications.ok(
+            <p>Extension <b>{displayName}</b> successfully uninstalled!</p>
+          );
+        });
+
+        removedUninstalling.forEach(({ id }) => {
+          this.extensionState.delete(id);
+        });
+      })
+    );
+  }
 
   @computed get extensions() {
     const searchText = this.search.toLowerCase();
     return Array.from(extensionLoader.userExtensions.values()).filter(ext => {
       const { name, description } = ext.manifest;
       return [
         name.toLowerCase().includes(searchText),
-        description.toLowerCase().includes(searchText),
-      ].some(v => v);
+        description?.toLowerCase().includes(searchText),
+      ].some(value => value);
```

Comment:
Would using `_.filter` be more readable?

Reply:
Maybe, but not going to touch unrelated code here.